infrastructure-playbooks/vars/lv_vars.yaml.sample

```diff
@@ -11,29 +10,42 @@
 # Having leftover signatures can result in ansible errors that say "device $device_name excluded by a filter" after running the lv-create.yml playbook.
 # This can be done by running `wipefs -a $device_name`.
 
-# Path of nvme device primed for LV creation for journals and data. Only one NVMe device is allowed at a time. Providing a list will not work in this case.
-nvme_device: dummy
-
-# Path of hdd devices designated for LV creation.
-hdd_devices:
-  - /dev/sdd
-  - /dev/sde
-  - /dev/sdf
-  - /dev/sdg
-  - /dev/sdh
+# nvme_device is the Path of nvme device primed for LV creation for journals/DB/WAL and data.
+# hdd_devices is a list of Path of hdd devices designated for LV creation.
+# lv_name var is a list of bucket index LVs created on the NVMe device. We recommend one be created but you can add others
+lvm_devices:
+  - nvme_device: dummy
+    hdd_devices:
+     - /dev/sdd
+     - /dev/sde
+     - /dev/sdf
+     - /dev/sdg
+     - /dev/sdh
+    nvme_device_lvs:
+     - lv_name: "ceph-bucket-index-1"
+       size: 100%FREE
+       journal_name: "ceph-journal-bucket-index-1-dummy }}"
```

Comment:
```suggestion
       journal_name: "ceph-journal-bucket-index-1-dummy"
```
typo?

Reply:
Hi, sorry for my slow response. yes, this was a typo